ee/tabby-db/src/repositories.rs

```diff
@@ -68,13 +83,13 @@ impl DbConn {
             .await?)
     }
 
-    pub async fn update_repository_url(&self, name: String, git_url: String) -> Result<bool> {
+    pub async fn update_repository_url(&self, id: i32, git_url: String) -> Result<bool> {
         Ok(self
             .conn
-            .call(|c| {
+            .call(move |c| {
                 let updated = c.execute(
-                    "UPDATE repositories SET git_url=? WHERE name=?",
-                    [git_url, name],
+                    "UPDATE repositories SET git_url=? WHERE id=?",
+                    (git_url, id),
                 )?;
                 Ok(updated == 1)
```

Comment:
When updated is not 1, shall we generate an error? or rusqlite already generate an error for us?

Reply:
SQL won't error if an update query affects no rows. I wasn't sure if that should be an error, do you think it would be preferable to returning false?